languages/scheme/src/main/java/de/jplag/scheme/SchemeLanguage.java

```diff
@@ -26,7 +26,7 @@ public String[] suffixes() {
 
     @Override
     public String getName() {
-        return "SchemeR4RS Parser [basic markup]";
+        return "SchemeR4RS [basic markup]";
```

Comment:
@tsaglam Is it wanted that we have `[basic markup]` here?

Reply:
This is a legacy thing, I am not even sure what exactly Scheme basic markup is. I am both fine with removing and leaving it.